packages/core/src/core.ts

```diff
@@ -2,8 +2,22 @@
  * Global JSONForms object that holds services and registries.
  */
 export class JsonFormsGlobal {
+  /**
+   * Array of renderers that should be picked up at startup.
+   * @type {any[]}
+   */
   public renderers = [];
+
+  /**
+   * Array of field renderers that should be picked up at startup.
+   * @type {any[]}
+   */
   public fields = [];
+
+  /**
+   * Object containing additional reducers.
+   * @type {{}}
```

Comment:
shouldn't type be the type of the method?

Reply:
I've removed the `reducers` property, since it is not use anyway.